src/js/editor/post.js

```diff
@@ -9,9 +11,24 @@ function isMarkupSection(section) {
   return section.type === MARKUP_SECTION_TYPE;
 }
 
+function isTextEditableSection(section) {
+  return isMarkupSection(section);
+}
+
+function isListItem(section) {
+  return section.type === LIST_ITEM_TYPE;
+}
+
+function isBlankAndLastListItem(section) {
+  return isListItem(section) &&
+    section.isBlank &&
+    section.parent.sections.tail.isBlank; // existing tail is blank (FIXME)
+}
+
 class PostEditor {
   constructor(editor) {
     this.editor = editor;
+    this.builder = this.editor.builder;
```

Comment:
Should update the docs to use `postEditor.builder` if we provide this. Seems better than

```
editor.run((postEditor) = {
  postEditor.insertSectionBefore(editor.builder.createMarkupSection('p'), editor.post.sections.head);
});
```

My main justification for hesitancy is that the builder is non-mutating. It just returns new things. But leaving it on the editor for public consumers encourages them to much with the post outside a `run` block, so I am :+1: on putting it on the `postEditor`.